RegressionUITests/RegressionUITests.swift

```diff
@@ -42,11 +42,12 @@ class RegressionUITests: XCTestCase {
     }
 
     // A helper method for keeping things cleaner when pushing to the dashboard, or saving a result locally.
-    func scanForAccessibility(withScanName: String = "unnamed scan") throws {
+    func scanForAccessibility(withScanName name: String = "unnamed scan") throws {
```

Comment:
I tried this out. Didn't work unfortunately.

Reply:
I just tested this branch out locally and was able to see the scan name and file name update. Was there any specific issue you were running into when testing on your end?

Reply:
@nbaddam can you share the json reports from your test run?

Reply:
[RegressionScans.zip](https://github.com/dequelabs/axe-devtools-ios-sample-app/files/12187351/RegressionScans.zip)

Reply:
This is what got saved to my machine after the test run!

Reply:
Hmm. The filename is fine, but the scanName property is not added to the json data.

Reply:
[axe-devtools-mobile-scans.zip](https://github.com/dequelabs/axe-devtools-ios-sample-app/files/12187458/axe-devtools-mobile-scans.zip)
@nbaddam here is what it should look like. I got these scan results from the mobile results dashboard, so it looks like `try axe?.postResult(result, withScanName: withScanName)` works as expected. Perhaps the issue is in `saveResult()`

Reply:
Oh interesting. Yeah the "scanName" and "tags" are only added to the json for scans pulled from the dashboard it looks like...odd

Reply:
The JSONS I linked still post to the dashboard with the correct scan name though. Once i export the same scan from the dashboard, I see the "scanName" and "tags" in the JSON.

Reply:
Okay, for now the scan name is also saved within the JSONS if you look for "axeMetaData" and then "screenTitle". Kim might have more insight on why the JSON formats are different and whether that can be fixed. Sorry for the confusion!

Reply:
we've got a fix for this sprint -- will be released soon.